pibooth/pictures/factory.py

```diff
@@ -273,9 +273,9 @@ def add_text(self, text, font_name, color, align=CENTER):
         assert align in [self.CENTER, self.RIGHT, self.LEFT], "Unknown aligment '{}'".format(align)
         self._texts.append((text, fonts.get_filename(font_name), color, align))
         if self.is_portrait:
-            self._texts_height = 600
+            self._texts_height = int(self.height * 0.167)
         else:
-            self._texts_height = 300
+            self._texts_height = int(self.height * 0.125)
```

Comment:
I would rather use int(self.height / 8) here, is there a reason to use * 0.125 instead?

Reply:
True